regions_www/m/pct_shiny_funs.R

```diff
@@ -275,15 +281,15 @@ popup_straight_lines <- function(data, scenario, purpose){
       <td>", round_dp(data[[data_filter(scenario, "sid")]]), "</td>
     </tr>
     <tr>
-      <td> Change in deaths/yr: &nbsp; </td>
-        <td style= 'color:", font_colour_death , "' >", round(data[[data_filter(scenario, "sideath")]], 3),
-           " (&pound;" , round(data[[data_filter(scenario, "sivalueyll")]]), ")
+      <td> Change in YLLs/yr: &nbsp; </td>
+        <td style= 'color:", font_colour_death , "' >", round(data[[data_filter(scenario, "siyll")]], 2),
+           " (&pound;" , pretty_num(round(data[[data_filter(scenario, "sivalueyll")]])), ")
```

Comment:
> 
> 
> @usr110 sure [4820fc6](https://github.com/npct/pct-shiny/commit/4820fc6c0f31211bd23c5023bf50c50eb61600f0) this adds an info box i.e.
> ![Screenshot from 2019-11-24 20-16-20](https://user-images.githubusercontent.com/6289830/69500911-8d62e100-0ef7-11ea-9a57-5856f00961de.png)

This is exactly what I had in mind. Brilliant, thanks a bunch.


> To make future change easier it might be easiest to do the `round` in the `pretty_num`?

Good point. Will incorporate it.



>  I am also seeing the repetition with the `data[[data_filter(scenario, "variable")]])` as something it'd be nice to clean up but that can wait for another PR.

Agreed.